src/libs/actions/User.js

```diff
@@ -0,0 +1,12 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+
+function getBetas() {
+    Onyx.merge(ONYXKEYS.BETAS, []);
+    API.User_GetBetas().then((response) => {
+        Onyx.set(ONYXKEYS.BETAS, response.betas);
```

Comment:
I might be paranoid, but I would add a check for `if (response.jsonCode === 200)` to avoid corrupting the data in Onyx.